### DISCUSSION TITLE
How to use the Connections with Mongodb?

### DISCUSSION THREAD

**Discussion by rtheodoro** [posit-dev/positron#5164] [Q&A]
Hi !

I want to know if its possible to use Connections tab  with Mongodb. Is there a tutorial?

Thanks a lot!

**Comment by sharon-wang** [comment]
Hey @rtheodoro!

Are you opening up a database connection with R? If you're already familiar with Connections in RStudio, your experience in Positron should be similar! You can refer to our docs on [Opening a connection using R](https://positron.posit.co/connections-pane.html#opening-a-connection-using-r) combined with some MongoDB-specific notes on Posit's Solutions site: https://solutions.posit.co/connections/db/databases/mongodb/. 

For [Python](https://positron.posit.co/connections-pane.html#opening-a-connection-using-python), we currently support connections created using the [sqlite3](https://docs.python.org/3/library/sqlite3.html) and [SQLAlchemy](https://www.sqlalchemy.org/) modules.

We're still building up our documentation, so if you have suggestions or run into any issues, please let us know!
